src/main/java/converter/XMLExporter.java

```diff
@@ -61,10 +61,7 @@ private void fillXMLWithParticipantsInfo(Map<String, Athlete> source, Map<String
 
     private Element addEventsResults(Map<String, Athlete> source, Document doc, Map.Entry<String, String> eventParticipants) {
         Element events = doc.createElement("events");
-        for (Map.Entry<String, String> participant : source.get(eventParticipants.getKey()).getEvents().entrySet()) {
-            Element eventPerformance = createTextNode(doc, participant.getKey(), participant.getValue() );
-            events.appendChild(eventPerformance);
-        }
+        source.get(eventParticipants.getKey()).getEvents().forEach((key, value) -> events.appendChild(createTextNode(doc, key, value)));
```

Comment:
Use new line on dot.
This line is really long and ir barely readable.
I prefer reduce in this case over forEach.